python/example_code/lambda/boto_client_examples/api_gateway_rest.py

```diff
@@ -0,0 +1,248 @@
+# Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+# SPDX-License-Identifier: Apache-2.0
+
+"""
+Purpose
+
+Shows how to use the AWS SDK for Python (Boto3) to use Amazon API Gateway to
+create a REST API backed by a Lambda function.
+
+Instead of using the low-level Boto3 client APIs shown in this example, you can use
+AWS Chalice to more easily create a REST API.
+
+    For a working code example, see the `lambda/chalice_examples/lambda_rest` example
+    in this GitHub repo.
+
+    For more information on AWS Chalice, see https://github.com/aws/chalice.
+"""
+
+import calendar
+import datetime
+import json
+import logging
+import time
+import boto3
+from botocore.exceptions import ClientError
+import requests
+
+import lambda_basics
+
+logger = logging.getLogger(__name__)
+
+
+def create_rest_api(
+        apigateway_client, api_name, api_base_path, api_stage,
+        account_id, lambda_client, lambda_function_arn):
+    """
+    Creates a REST API in Amazon API Gateway. The REST API is backed by the specified
+    AWS Lambda function.
+
+    This function takes several steps to put the pieces together:
```

Comment:
Maybe, "The following is how the function puts the pieces together"? Are these done in order? if that's so, "... together, in order"?